plugins/org.jboss.tools.openshift.cdk.server/src/org/jboss/tools/openshift/internal/crc/server/core/adapter/CRC100Server.java

```diff
@@ -77,10 +77,6 @@ public boolean isInitialized() {
 		if (!bin.exists() || !json.exists() || !log.exists()) {
 			return false;
 		}
-		File oc = new File(bin, "oc");
```

Comment:
This check looks good to me but it is Windows only. It should be adapted rather than deleted

Reply:
This is really quite a check I have to say, it works for both windows and linux but it is because in reality it checks existence of folder `oc` in directory `bin` which is there for linux and windows. `oc.exe` file exists under this `oc` directory. so I would say check for file `oc.exe` is useless in this case.